src/cli.zig

```diff
@@ -1636,10 +1636,12 @@ pub const Command = struct {
             // if we are bunx, but NOT a symlink to bun. when we run `<self> install`, we dont
             // want to recursively run bunx. so this check lets us peek back into bun install.
             if (args_iter.next()) |next| {
-                if (bun.strings.eqlComptime(next, "add") and
-                    bun.getenvZ("BUN_INTERNAL_BUNX_INSTALL") != null)
-                {
-                    return .AddCommand;
+                if (bun.getRuntimeFeatureFlag("BUN_INTERNAL_BUNX_INSTALL")) {
```

Comment:
the original ordering of these operations was intentional. the environment variable shouldn't get queried if the command doesn't match. can we do the lame code dupe and have two if statements that do `if (eqlComptime(next, "...") and env check)`

Reply:
cool cool, i wasn't the perf and if runtime feature flag was more efficient (should i still use it as it appears what most new stuff checking env var is using?)